client/app/shared/ace-editor/ace-editor.directive.js

```diff
@@ -0,0 +1,43 @@
+/** @ngInject */
```

Comment:
ditto that feel simply because I would love to see onDestroy used here... once we're not using ACE to clean up nicely... or you can do a destroy of the directive once its use has expired i suppose...